umbraco/Umbraco-CMS
# Fix has pending package migrations

### Description
Fixes issue, where packages with package migration did report "hasPendingMigrations" when it did not.

The issue seems to happen, when the package Name is different from the package Id.

### Tests
- Install Umbraco.Licenses to Umbraco.Web.UI
- Boot
- Go swagger and use the management api as the client also has issues
- Check the `/umbraco/management/api/v1/package/migration-status` endpoint and verify there is no pending if the package migrations are executed on boot.
